packages/salesforcedx-sobjects-faux-generator/src/messages/i18n.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * Conventions:
+ * _message: is for unformatted text that will be shown as-is to
+ * the user.
+ * _text: is for text that will appear in the UI, possibly with
+ * decorations, e.g., $(x) uses the https://octicons.github.com/ and should not
+ * be localized
+ *
+ * If ommitted, we will assume _message.
+ */
+export const messages = {
+  faux_generation_cancelled_text: 'Faux class generation cancelled',
```

Comment:
Our style guidelines say to use only one 'l' in "canceled": https://confluence.internal.salesforce.com/display/public/DOCSTYLE/canceled%2C+canceling%2C+cancellation